R/generate_simulated_data.R

```diff
@@ -50,6 +50,15 @@
 #' LOD across sites
 #' @param input_params_path path to the toml file with the parameters to use
 #' to generate the simulated data
+#' #' @param corr_function Correlation function for spatial site correlations
+#' @param corr_fun_params Parameters required for correlation function
```

Comment:
Would add what the default is and its implications

Reply:
Alternatively, I think we could and probably should just create an boolean switch that indicates whether or not to generate spatially correlated data, so we could add an argument that is `@param include_spatial_corr = TRUE` and then run those components if TRUE and not if FALSE?